src/core/friendly_errors/fes_core.js

```diff
@@ -142,7 +142,11 @@ if (typeof IS_MINIFIED !== 'undefined') {
 
       const funcName =
         methodParts.length === 1 ? func : methodParts.slice(2).join('/');
-      msgWithReference = `${message} (http://p5js.org/reference/#/${referenceSection}/${funcName})`;
+
+      //Whenever func having p5.[Class] is encountered, we need to have the error link as mentioned below else different link
+      funcName.substring(0,2) === 'p5.' ?
```

Comment:
```js
funcName.startsWith('p5.') ?
```

Reply:
oh wow! when was [this method ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) added to JS strings? don't think i'd seen it before. thnx @wuyudi  

it's not completely necessary, but it would be easier for other contributors to read/understand, what do u think @Ayush23Dash

Reply:
We can use this @nbriz! Thanks @wuyudi, Ill just update the PR